lib/linguist/languages.yml

```diff
@@ -303,6 +305,7 @@ Bison:
   extensions:
   - .bison
   ace_mode: text
+  color: "#6A463F"
```

Comment:
@pchaigno Bison is not the same with Yacc, just similar. Maybe, in the future, Bison's syntax becomes more unique and Github starts to differentiate Bison and Yacc for its stats bar. =) Also I think stable colors better than dinamic ones, can anything be done about that?